LICENSE

```diff
@@ -1,23 +1,47 @@
-The MIT License (MIT)
```

Comment:
I notice this doesn't explicitly call out `MIT License`. Is there a place for that. I've had folks in the past not know off hand that the license is MIT without the little blurb.